airflow/providers/cncf/kubernetes/operators/pod.py

```diff
@@ -781,8 +781,9 @@ def write_logs(self, pod: k8s.V1Pod):
                 follow=False,
             )
             for raw_line in logs:
-                line = raw_line.decode("utf-8", errors="backslashreplace").rstrip("\n")
-                self.log.info("Container logs: %s", line)
+                line = raw_line.decode("utf-8", errors="backslashreplace").strip()
```

Comment:
Is there any specific reason to update the strip method? If the user log something similar to Airflow logo in our CLI, the result will be completely different. Currently we strip new line because the logger log each message on a new line.

Reply:
sometimes this log like
```
Container logs: 
```
but since I'm checking `if line` I should be able to revert this change and that way at least it will not log empty("") str wdyt?